CMakeLists.txt

```diff
@@ -93,6 +93,8 @@ if(NOT DNNL_TARGET_ARCH)
         set(DNNL_TARGET_ARCH "PPC64")
     elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "^(s390x.*|S390X.*)")
         set(DNNL_TARGET_ARCH "S390X")
+    elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "^(rv.*|RV.*|riscv.*|RISCV.*)")
```

Comment:
Shell we also filter-out riscv32?

Reply:
I don't think this is necessary because of this code above that validates we're using a 64-bit architecture:

```cmake
if (NOT CMAKE_SIZEOF_VOID_P EQUAL 8)
    message(FATAL_ERROR "oneDNN supports 64 bit platforms only")
endif()
```

Reply:
My bad, resolving.